CreateClothingPRP.py

```diff
@@ -285,17 +286,20 @@ def CreatePage(input_path: Path, output_path: Path, gcAgeInfo: plAgeInfo, pageIn
     plResMgr.WritePage(filename, newPage)
     return newPage.location
 
-def ProcessClothingJSON(input_path: Path, output_path: Path, gcAgeInfo: plAgeInfo) -> Generator[plLocation]:
+def ProcessClothingJSON(input_path: Path, output_path: Path, gcAgeInfo: plAgeInfo, singlePage: Union[None, str] = None) -> Generator[plLocation]:
     print("Loading clothing instruction file...")
     with input_path.open("r") as clothingFile:
         clothingData = json.load(clothingFile)
         pages = clothingData["pages"]
```

Comment:
Let's do the filtering logic separately from the processing logic.

```suggestion
        pages = clothingData["pages"]
        if single_page:
            pages = (i for i in pages if i["name"] == single_page)
```

Then remove the if-continue from the loop.